Header header
string frame_prefix
MarkerPose[] markers